x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/scheduler/Cron.java

```diff
@@ -232,6 +232,8 @@ public class Cron implements ToXContentFragment {
 
     private final String expression;
 
+    private TimeZone timeZone;
```

Comment:
We probably ought to include this in equals/hashCode now, right?

Reply:
Yes good spot. I'll get that added :-)